validator/testcases/regex.py

```diff
@@ -567,6 +568,20 @@ def tests(self):
             compat_type='error')
 
 
+@register_generator
+class Gecko54RegexTests(CompatRegexTestHelper):
+    VERSION = FX54_DEFINITION
+
+    def tests(self):
+        yield self.get_test_bug(
+            1333482,
+            r'-moz-appearance\b',
+            '-moz-appearance can only be used in chrome stylesheets.',
+            'All other users will be ignored.',
```

Comment:
I don't understand what this sentence is trying to say. Do you mean `uses`? Can we rephrase to `Any other usage ...` or something like that?